kdump/include/runtest.sh

```diff
@@ -295,7 +298,7 @@ RhtsSubmit() {
 GetBiosInfo()
 {
     # Get BIOS information.
-    dmidecode >"${K_TMP_DIR}/bios.output"
+    which dmidecode && dmidecode >"${K_TMP_DIR}/bios.output"
```

Comment:
Do we need to make sure package `dmidecode` is installed via Makefile? 
OR just keep the checking but don't upload file ${K_TMP_DIR}/bios.output if dmidecode is missing?

Reply:
I don't want to rely too much on beaker metadata/Makefile support. But you are right, I probably need to try installing the package first. Then decide whether to run dmidecode to generate bios.output. 
But still, the old code is harmless actually except generating an empty useless file. rhts-submit-log will not upload any empty file.
I will fix it.

Reply:
Hi @idorax, 
I fixed the dmidecode check. Can you please have a look?

Reply:
Hi @emmawu, I didn't know that rhts-submit-log doen't upload empty file as a matter of fact, sorry for that. Your fix looks good to me, thanks!